Cargo.toml

```diff
@@ -31,6 +31,7 @@ dirs = "5.0.1"
 # Otherwise any changes to the local files will not reflect in the compiled
 # binary.
 kanata-keyberon = { path = "keyberon" }
+regex = "1.8.4"
```

Comment:
I would prefer not to add regex as a dependency. It's an excellent crate, but the extra dependency does increase compile times, and increases the stripped binary size by about 900 KB, up from 1.4MB to 2.3MB for the default Linux binary.

It's probably a minor/negligible thing for most, but since I think the processing being done with the regex is simple enough to do by other string operations, it seems better to go with those.

Reply:
That's totally fine with me and I can make the change, but does it change your mind to know that kanata already implicitly uses the `regex` crate? If you look at the diff of [`Cargo.lock`](https://github.com/jtroo/kanata/pull/456/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87e) you'll see the only real change this makes to the lock file is that regex goes from `1.8.2` to `1.8.4`.

Reply:
Ah good point! I wonder if the transitive dependency removes some of the default features? It's strange to me that the binary size increases quite a lot between this PR and the current main branch.

Reply:
Maybe, or maybe it's because the version I added is different from the one already used implicitly. I wonder if pinning it to 1.8.2 would make the binary smaller?